users/static/users/center_style.css

```diff
@@ -433,3 +433,7 @@ span {
     padding: 2%;
     float: left;
 }
+
+#top-table-part {
```

Comment:
It shouldn't be id, but class, as it is repeatable multiple times. If you want an id, create it for whole table, as it is used once, ie `#notification-table` and access to cells using xpath like `#notification-table thead tr td`.

Reply:
Done